dev/tools/update-compat.py

```diff
@@ -305,22 +373,38 @@ def parse_args(argv):
     args = {
         'assert_unchanged': False,
         'cur_version': None,
-        'number_of_old_versions': DEFAULT_NUMBER_OF_OLD_VERSIONS
+        'number_of_old_versions': None,
+        'master': False,
+        'release': False,
+        'git_add': False,
     }
+    if '--master' not in argv and '--release' not in argv:
+        print(r'''WARNING: You should pass either --release (right before branching)
+  or --master (right after branching and updating the version number in version.ml)''')
```

Comment:
```suggestion
  or --master (right after branching and while updating the version number in version.ml)''')
```

Reply:
It is important that the version number in `version.ml` is updated *before* running the script.  "while" doesn't convey this to me.

Reply:
I see.